src/main/client/featureToggleClient.ts

```diff
@@ -0,0 +1,45 @@
+import { CoreOptions, RequestAPI } from 'request'
+import { RequestPromise } from 'request-promise-native'
+
+import { HeadersBuilder } from './headersBuilder'
+
+export class FeatureToggleClient {
+  private endpointURI: string
+  private request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>
+
+  constructor (endpointURI: string,
```

Comment:
We can do:

```
constructor (private endpointURI: string,
             private request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>) {
```

This way we will be able to remove:

```
private endpointURI: string
private request: RequestAPI<RequestPromise, CoreOptions, CoreOptions>
```

Reply:
Code updated as suggested :+1: